dmcontent/questions.py

```diff
@@ -634,10 +634,31 @@ def value(self):
 
     @property
     def answer_required(self):
+        """Checks all sub-questions and returns true if any questions which are required, still require answers.
+        Appropriately checks/ignores followup questions based on current answers."""
         if self.get('optional'):
             return False
 
-        return any(question.answer_required for question in self.questions)
+        lookup_question_by_id = {q.id: q for q in self.questions}
+        followup_questions = set()
+
+        # For each question that has followups, check the followup questions - if any require answers, the multiq does.
+        for question in self.questions:
+            answer = question.value if isinstance(question.value, list) else [question.value]
```

Comment:
I think it might make more sense if this variable was plural-named, seeing as its a list. Maybe 'answers' or 'selected_values' or something?